core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -611,6 +611,8 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
           CoreUtils.swallow(zkUtils.close())
         if (metrics != null)
           CoreUtils.swallow(metrics.close())
+        if (transactionCoordinator != null)
+          CoreUtils.swallow(transactionCoordinator.shutdown())
```

Comment:
This needs to be removed as we have already merged a PR that fixes this:

https://github.com/apache/kafka/commit/3085d4f435bba74fc9aa077e23a4da3475b4d2ec

Note that the shutdown needs to happen before the LogManager is shutdown (we had an issue where the GroupCoordinator tried to write to an already closed LogManager in the past).